docs/src/pages/en/media-controller.md

```diff
@@ -8,31 +8,17 @@ The `<media-controller/>` manages communication of state and state change reques
 
 ## Attributes
 
-- `autohide (seconds, default: 0)` - (`video` only) Use this to autohide all controls/chrome (except for the `media`) after `n` seconds of inactivity, unless the media is paused. To disable `autohide`, set the value to -1.
+### autohide
```

Comment:
idea(non-blocking): As a followup, especially if we're pairing down the examples, I'm wondering if we should just make the `media-controller` documentation look more like our other Media State Receiver components (i.e. just tables) for consistency.

Reply:
I agree. Gonna save that for a separate pass.